src/js/view/controls/controlbar.js

```diff
@@ -305,6 +306,15 @@ export default class Controlbar {
         _model.change('fullscreen', (model, val) => {
             const fullscreenElements = [this.elements.fullscreen.element(), this.elements.imaFullscreen.element()];
 
+            if (fullscreenElements[0].getAttribute('aria-label') === 'Exit Fullscreen') {
+                // Uses setTimeout to prevent 'Exit Fullscreen Successful' voice line from being cutoff by other voice lines when using a screen reader
+                setTimeout(() => {
+                    this._exitFullscreenAnnouncer.textContent = 'Exit Fullscreen Successful';
+                }, 1450);
+            } else if (fullscreenElements[0].getAttribute('aria-label') === 'Fullscreen') {
+                this._exitFullscreenAnnouncer.textContent = '';
```

Comment:
We should probably add an Enter fullscreen text when it's entered too. Also, is there any reference to what the element is when VoiceOver reads the exit success message? If not we should specify in the message, i.e.: "Player is now fullscreen", "Player is no longer fullscreen"